src/async_impl/client.rs

```diff
@@ -156,6 +160,22 @@ impl ClientBuilder {
 
                     Connector::new_default_tls(tls, proxies.clone(), config.local_address, config.nodelay)?
                 },
+                #[cfg(feature = "default-tls")]
+                TlsBackend::BuiltDefault(mut conn) => {
+                    Connector::from_built_default(
+                            std::mem::replace(&mut conn, native_tls::TlsConnector::new().unwrap()),
```

Comment:
Can you not move it here?

Reply:
No, the compiler doesn't accept the mutable reference

Reply:
What mutable reference?

Reply:
Oh I see, `replace(conn, ...` instead of `replace(&mut conn, ...`?

Reply:
No get rid of the `replace` altogether.

Reply:
Right right, I thought we were talking about `src/client.rs` not `src/async_impl/client.rs`